telemetry/influxdb/src/main/scala/io/buoyant/telemetry/influxdb/InfluxDbTelemeter.scala

```diff
@@ -0,0 +1,118 @@
+package io.buoyant.telemetry.influxdb
+
+import com.twitter.finagle.Service
+import com.twitter.finagle.http.{MediaType, Request, Response}
+import com.twitter.finagle.stats.NullStatsReceiver
+import com.twitter.finagle.tracing.NullTracer
+import com.twitter.util.{Awaitable, Closable, Future}
+import io.buoyant.admin.Admin
+import io.buoyant.telemetry.{Metric, MetricsTree, Telemeter}
+
+/**
+ * This telemeter exposes metrics data in the InfluxDb LINE format, served on
+ * an admin endpoint.  It does not provide a StatsReceiver or Tracer.  It reads
+ * histogram summaries directly off of the MetricsTree and assumes that stats
+ * are being snapshotted at some appropriate interval.
+ */
+class InfluxDbTelemeter(metrics: MetricsTree) extends Telemeter with Admin.WithHandlers {
+
+  private[influxdb] val handler = Service.mk { request: Request =>
+    val response = Response()
+    response.version = request.version
+    response.mediaType = MediaType.Txt
+    val sb = new StringBuilder()
+    val host = request.host.getOrElse("none")
```

Comment:
does this write the host header of the request to the metrics endpoint as a tag?  influx is weird.